### ISSUE TITLE
Separate dependency changes from other changes

### ISSUE THREAD

**Issue by kolaente** [unjs/changelogen#74]
### Describe the feature

I'm using renovate in my project to manage dependencies (as does this one). Renovate generates commit messages like `fix(deps): update dependency ...`. I would like to group these dependency changes not under the fixes section but in a separate group. Usually there are a lot of dependency updates between releases and thus they clutter the changelog.

I can look into implementing this if someone points me in the right direction.

### Additional information

- [X] Would you be willing to help implement this feature?

**Comment by pi0** [comment]
Good idea. I think we might introduce a new option like `typeGroups` or `scopeGroup`, mapping scope (which is `deps`) to a new category in generated changelog.
